ibis/backends/datafusion/tests/conftest.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+
+import pyarrow as pa
+
+import ibis
+import ibis.expr.types as ir
+from ibis.backends.tests.base import BackendTest, RoundAwayFromZero
+
+
+class TestConf(BackendTest, RoundAwayFromZero):
+    # check_names = False
+    # additional_skipped_operations = frozenset({ops.StringSQLLike})
+    # supports_divide_by_zero = True
+    # returned_timestamp_unit = 'ns'
+    bool_is_int = True
+
+    @staticmethod
+    def connect(data_directory: Path):
+        # can be various types:
+        #   pyarrow.RecordBatch
+        #   parquet file path
+        #   csv file path
+        client = ibis.datafusion.connect({})
+        client.register_csv(
```

Comment:
I wonder if we should deprecate the csv, hdf5 and parquet "backends" and have a standard system to register files that can be used in the pandas backend, as you are doing here.

Reply:
This seems like a good idea. There are also a growing number existing and potential backends that support local file consumption: datafusion, duckdb, pyarrow, clickhouse, etc.